app/views/layouts/application.html.slim

```diff
@@ -2,6 +2,7 @@ doctype html
 html
   head
     title Codestats
+    = favicon_link_tag 'codestats_favicon.png', type: 'image/png'
```

Comment:
it should if you want to support all legacy browsers, but all modern browsers (tested in safari, opera, firefox, chrome) also support png files which are much simpler to create and modify.